inductiva/utils/optional_deps.py

```diff
@@ -50,14 +50,33 @@ def wrapper(*args, **kwargs):
     return wrapper
 
 
-molecules_missing_deps_msg = (
-    "You can install the missing dependencies for 'molecules' "
-    "with `pip install 'inductiva[molecules_extra]'`.")
+def _missing_deps_msg(extra_deps_name):
+    return (f"You can install the missing dependencies for '{extra_deps_name}' "
+            f"with `pip install 'inductiva[{extra_deps_name}_extra]'`.")
+
 
 # Apply the two first arguments to _needs_optional_deps function, creating
 # a new decorator function with those arguments already set.
 needs_molecules_extra_deps = functools.partial(
     _needs_optional_deps,
-    ["MDAnalysis", "nglview"],
-    molecules_missing_deps_msg,
+    ["MDAnalysis", "nglview", "imageio", "matplotlib"],
+    _missing_deps_msg("molecules"),
+)
+
+needs_fluids_extra_deps = functools.partial(
+    _needs_optional_deps,
+    ["pyvista", "vtk", "scipy", "xarray", "dask", "imageio", "matplotlib"],
+    _missing_deps_msg("fluids"),
+)
+
+needs_coastal_extra_deps = functools.partial(
+    _needs_optional_deps,
+    ["utm", "imageio", "scipy", "matplotlib"],
```

Comment:
imageio and matplotlib are already in the common extra, if I understand correctly your mechanism they are not needed here, right?

Reply:
Or is it for the warning when the import fails?

Reply:
It is for the purpose of the warning, and mention that this will be the ones needed to install.